x-pack/plugins/rule_registry/server/rule_data_client/rule_data_client.ts

```diff
@@ -105,11 +105,17 @@ export class RuleDataClient implements IRuleDataClient {
     return {
       search: async (request) => {
         const clusterClient = await waitUntilReady();
+        let body;
```

Comment:
Nit: Can't we wrap the entire `search` function with the `try/catch` block, by doing that we would
- Make it easier to read as the `return` statement will be inside the `try`.
- Avoid defining `body`.
- Will catch any error thrown by`waitUntilReady()`.

Reply:
Great idea! Updated in https://github.com/elastic/kibana/pull/136225/commits/83e971e1155e43b91414e940ec779754cc61c9af